opsdroid/connector/webexteams/__init__.py

```diff
@@ -0,0 +1,107 @@
+"""A connector for Webex Teams."""
+import json
+import logging
+import uuid
+
+import aiohttp
+
+from ciscosparkapi import CiscoSparkAPI
```

Comment:
Are they going to rename this at some point? We may want to raise an issue to just keep an eye on that as we may have to do this at a later date.

Reply:
I will raise an issue for it after merging this pr